### DISCUSSION TITLE
so-elasticsearch-templates-load modify

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#7398] [Ideas] [answered]
Hi!
so-elasticsearch-templates-load has such code:
`for i in *; do TEMPLATE=$(echo $i | cut -d '-' -f2); echo "so-$TEMPLATE"; curl -K /opt/so/conf/elasticsearch/curl.config -k ${ELASTICSEARCH_AUTH} -s -XPUT -L https://${ELASTICSEARCH_HOST}:${ELASTICSEARCH_PORT}/_template/so-$TEMPLATE -H 'Content-Type: application/json' -d@$i 2>/dev/null; echo; done`
It's working correctly with templates that called with mask ie `so-source-template`. But if template named like `so-client-source-template` it causes error. I made workaround by modifying string:
`for i in *; do parts=$(echo $i | grep -o "-" | wc -l); echo $parts;  i=$(echo $i | cut -d '.' -f1); TEMPLATE=$(echo $i | cut -d '-' -f$(seq -s, 2 1 $parts )); echo "so-$TEMPLATE"; curl -K /opt/so/conf/elasticsearch/curl.config -k ${ELASTICSEARCH_AUTH} -s -XPUT -L https://${ELASTICSEARCH_HOST}:${ELASTICSEARCH_PORT}/_template/so-$TEMPLATE -H 'Content-Type: application/json' -d@$i 7>/dev/null; echo; done`
It covers everything by mask so-1-2-3-N-template.json
Question is: may I create a PR for this?

**Comment by weslambert** [comment, marked as answer]
Yes, please send a PR and we can review it.
